src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Description>OpenTelemetry protocol exporter over gRPC for OpenTelemetry .NET</Description>
+    <PackageTags>$(PackageTags);OTLP;gRPC</PackageTags>
+    <MinVerTagPrefix>core-</MinVerTagPrefix>
+  </PropertyGroup>
```

Comment:
For new projects I would really like us to turn on things like...

```xml
    <Nullable>enable</Nullable>
    <AnalysisLevel>latest-all</AnalysisLevel>
    <ImplicitUsings>enable</ImplicitUsings>
```

...but we're probably going to be copying in a lot of stuff here from the existing project?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3924